### PR TITLE
Add Jira styles

### PR DESCRIPTION
**Pull request by ackernaut**
In support of #2773 

- [x] Updates styles for menus including card integration options and search
- [x] Updates styles for panels/rows including provider list and details under team settings
- [x] Includes housekeeping to reduce usage of `ui.js`
- [x] Updates buttons/panels for organization, notifications, and user profile views

### Test
Everything looks as expected per latest designs/theme:
- [ ] Add provider via card menu
- [ ] Integrate a card via menu
- [ ] Interact with provider list under team settings
- [ ] Interact with provider details under team settings
- [ ] Interact with notifications
- [ ] Interact with organization billing and members
- [ ] Interact with user profile

### CODE REVIEW

**Comment by ackernaut** [comment]
I think this is ready for your eyes @mattkrick 

There is one nuance that I found that I tested with and without style changes and it seems to be there either way. If a user blurs the menu search control for integrations they can’t focus on the input again. It’s tricky to do but I did it to test styling of the default border color. If you get a menu with only a few items, there is the specified gutter (8px) after the last item. You can click there to blur the input.

File: src/universal/components/AddToGitHubMenuItem.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ const AddToGitHubMenuItem = forwardRef((props: Props, ref) => {
       label={
         <MenuItemLabel>
           <MenuItemComponentAvatar>
-            <GitHubSVG />
+            <GitHubSVG18 />
```
Comment (mattkrick) [inline src/universal/components/AddToGitHubMenuItem.tsx]: Why are we constraining a DOM SVG to a particular size? We should instead style the svg like any other element

File: src/universal/components/Panel/Panel.js [hunk 2]
```diff
@@ -1,52 +1,43 @@
...
-  lineHeight: ui.labelHeadingLineHeight,
-  padding: `.75rem ${compact ? ui.panelCompactGutter : ui.panelGutter}`,
+const PanelLabel = styled(LabelHeading)(({compact}) => ({
+  padding: `12px ${compact ? ROW_GUTTER_COMPACT : ROW_GUTTER}px`,
   textTransform: 'uppercase'
 }))
 
 const PanelControls = styled('div')(({compact}) => {
-  const padding = compact ? ui.panelCompactGutter : ui.panelGutter
+  const padding = compact ? ROW_GUTTER_COMPACT : ROW_GUTTER
```
Comment (mattkrick) [inline src/universal/components/Panel/Panel.js]: looks like the compact variable isn't used. remove?

File: src/universal/components/Row/RowInfoCopy.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import styled from 'react-emotion'
+import {ROW_SUBHEADING_STYLES} from 'universal/styles/rows'
+
+const RowInfoCopy = styled('div')({
+  ...ROW_SUBHEADING_STYLES
```
Comment (mattkrick) [inline src/universal/components/Row/RowInfoCopy.ts]: lets keep those styles inside this component & then if another component needs to based off of this one we can use `RowInfoCopy.withComponent`

File: src/universal/components/SuggestedIntegrationGitHubMenuItem.tsx [hunk 4]
```diff
@@ -20,6 +21,12 @@ interface Props {
   query: string
 }
 
+const BlockSVG = styled('div')({
```
Comment (mattkrick) [inline src/universal/components/SuggestedIntegrationGitHubMenuItem.tsx]: this div looks unnecessary, can we apply the styles to MenuItemComponentAvatar?

File: src/universal/modules/teamDashboard/components/ProviderRow/ProviderRowName.tsx [hunk 5]
```diff
@@ -1,64 +1,70 @@
-import React from 'react'
```
Comment (mattkrick) [inline src/universal/modules/teamDashboard/components/ProviderRow/ProviderRowName.tsx]: delete?

File: src/universal/styles/providers.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+// for provider UI components
+
+export const GITHUB_NAME = 'GitHub'
```
Comment (mattkrick) [inline src/universal/styles/providers.js]: let's put all these inside `universal/constEnums.d.ts` under an enum like `Provider`.
That way, the compiled code doesn't have to import anything, the references are just replaced by the values

File: src/universal/styles/rows.ts [hunk 7]
```diff
@@ -0,0 +1,13 @@
+// rows.js
+
+import {PALETTE} from 'universal/styles/paletteV2'
+
+export const ROW_BORDER_COLOR = PALETTE.BORDER.LIGHTER
+// TODO: define generic gutter values for all components (TA)
+export const ROW_GUTTER = 16
+export const ROW_GUTTER_COMPACT = 12
+export const ROW_SUBHEADING_STYLES = {
```
Comment (mattkrick) [inline src/universal/styles/rows.ts]: let's move this into the component itself (see other comment)

File: src/universal/styles/rows.ts [hunk 8]
```diff
@@ -0,0 +1,13 @@
+// rows.js
+
+import {PALETTE} from 'universal/styles/paletteV2'
+
+export const ROW_BORDER_COLOR = PALETTE.BORDER.LIGHTER
```
Comment (mattkrick) [inline src/universal/styles/rows.ts]: let's remove this & just let components reference `PALETTE.BORDER.LIGHTER`, that way they don't need to import anything, the value just shows up.

File: src/universal/styles/rows.ts [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+// TODO: define generic gutter values for all components (TA)
+export const ROW_GUTTER = 16
```
Comment (mattkrick) [inline src/universal/styles/rows.ts]: let's move the gutter values to `constEnums` (maybe under a new GUTTERS enum?)

File: src/universal/styles/ui.js [hunk 10]
```diff
@@ -483,7 +480,7 @@ const ui = {
   // Menus
   // ---------------------------------------------------------------------------
   menuBackgroundColor: '#fff',
-  menuBorderColor: appTheme.palette.mid30l,
+  menuBorderColor: '#DFDFE5',
```
Comment (mattkrick) [inline src/universal/styles/ui.js]: looks like this is only used once. can we remove it from ui.js and replace with BORDER.LIGHTER?

**Comment by ackernaut** [comment]
@mattkrick these requests are very helpful! One thing I was navigating: it seems that when a TS files exports the `ENUM` the file importing it _has_ to be either `.ts` or `.tsx` instead of `.js`? I think I’ll have to format some component files to declares types when adding the suggested changes to use `ENUM`. Does that make any sense? I haven’t been in TS land for a second so I need a pointer for sure!

**Comment by mattkrick** [comment]
That's right, it's gotta be a typescript file to use enums, so we'll need to change the extension for those older components

File: src/universal/modules/teamDashboard/components/ProviderRow/ProviderRowName.tsx (hunk 5 above)
Comment (ackernaut) [reply src/universal/modules/teamDashboard/components/ProviderRow/ProviderRowName.tsx]: Will do, I think in #2740 we remove `userCount` and `integrationCount` as I’m wanting to simplify the design and not have those icons anymore.

**Comment by ackernaut** [comment]
@mattkrick I’ve addressed the feedback comments. Sorry this PR got so big with the housekeeping!

Converting these to TS has unleashed some new warnings. I think the RetroTemplateListMenu already had the warning.
- Panel.tsx
- TeamSettings.tsx
- Notifications.tsx
- AddSlackChannel.tsx
- RetroTemplateListMenu.tsx

Could you take a look at those and see if they are easy fixes?

File: src/universal/styles/paletteV2.ts [hunk 11]
```diff
@@ -16,9 +16,11 @@ export namespace PALETTE {
   }
 
   export const enum BORDER {
-    LIGHT = '#D1CBDB',
+    LIGHTER = '#DFDFE5',
+    LIGHT = '#C1C0CD',
     DARK = '#b6adc7',
-    MAIN = '#493272'
+    MAIN = '#493272',
+    BLUE_RGB = '50, 154, 229'
```
Comment (mattkrick) [inline src/universal/styles/paletteV2.ts]: can we just bake the alpha into the hex & use that instead? eg `#329AE580`. That way, the alpha is static so i won't be encouraged to play around with the alpha & make slightly wrong colored blues.

**Comment by mattkrick** [comment]
ts warnings fixed

**Comment by ackernaut** [comment]
ty looking now!

**Comment by ackernaut** [comment]
Ok, ready!

**Comment by mattkrick** [comment]
### Test
Everything looks as expected per latest designs/theme:
- [ ] Add provider via card menu
- [ ] Integrate a card via menu
- [ ] Interact with provider list under team settings
- [ ] Interact with provider details under team settings
- [ ] Interact with notifications
- [ ] Interact with organization billing and members
- [ ] Interact with user profile

**Pull request merged**